main.go

```diff
@@ -201,8 +238,20 @@ func getRepo(directory string, credentials []IssueAPI) (string, IssueAPI, error)
 			"associated with it.")
 	}
 
+	aliases, err := getURLAliases()
+	if err != nil {
+		return "", nil, err
```

Comment:
This aborts the whole application if there is no `~/.gitconfig`. This behavior is not right. If there is no `.gitconfig` we should just use an empty alias table to resolve URLs.

Reply:
That's right. How should I fix this? I have two ideas:

Remove error returning from `getURLAliases()` and leave only returning empty map, so user will see "xyz does not match any of the hosts" on error.

OR

In those 2 snippets from `getURLAliases()` return `nil` instead of function error

```golang
usr, err := user.Current()
if err != nil {
	return map[string]string{}, err
}
```

```golang
cfg, err := ini.Load(path)
if err != nil {
	return map[string]string{}, err
}
```

and in snippet below instead of returning `err` - return error with text like "Error while resolving alias for xyz"

```golang
for _, elem := range urlSections {
	section := cfg.Section(elem[0])
	alias, err := section.GetKey("insteadOf")
	if err != nil {
		return map[string]string{}, err
	}

	aliases[alias.Value()] = elem[1]
}
```

In this case when something bad happened while trying to get username or opening gitconfig user will see "xyz does not match any of the hosts" and when error occurs in parsing INI, user will see "Error while resolving alias for xyz"

Reply:
lol, if getURLAliases returned empty map and no error it would just work as expected.

Reply:
Okay, I fixed it